test/suites/ami/ami_test.go

```diff
@@ -12,32 +12,59 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package integration_test
+package ami_test
 
 import (
+	"testing"
+
+	awssdk "github.com/aws/aws-sdk-go/aws"
+
+	corev1beta1 "sigs.k8s.io/karpenter/pkg/apis/v1beta1"
+
 	"encoding/base64"
 	"fmt"
 	"os"
 	"strings"
 	"time"
 
-	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/karpenter-provider-aws/pkg/apis/v1beta1"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
 	"github.com/aws/aws-sdk-go/service/ec2"
 	"github.com/samber/lo"
 	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/types"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
+	"github.com/aws/karpenter-provider-aws/test/pkg/environment/aws"
+
 	coretest "sigs.k8s.io/karpenter/pkg/test"
+)
 
-	corev1beta1 "sigs.k8s.io/karpenter/pkg/apis/v1beta1"
+var env *aws.Environment
```

Comment:
Consider changing this to be consistent with our conversion
```suggestion
var env *environmentaws.Environment
```